app/policies/user_subscription_policy.rb

```diff
@@ -0,0 +1,15 @@
+class UserSubscriptionPolicy < ApplicationPolicy
+  def create?
+    user.present?
+  end
+
+  def destroy?
+    create? && [record.subscriber_id, record.subscription_id].include?(user.id)
+  end
+
+  class Scope < Scope
```

Comment:
Irrational creation of `Scope` class in each model policy. I guess that we can restrict ourselves to the implementation of this class in the `ApplicationPolicy`